src/components/NavItem/NavItem.styled.ts

```diff
@@ -0,0 +1,78 @@
+/* Internal dependencies */
+import { styled, css, ellipsis } from 'Foundation'
+import { InterpolationProps } from 'Types/Foundation'
+import { Icon } from 'Components/Icon'
+
+interface WrapperProps extends InterpolationProps {
+  active: boolean
+  open: boolean
+  paddingLeft: number
+}
+
+export const LeftIconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-right: 10px;
+`
+
+export const ContentWrapper = styled.div`
+  ${ellipsis()}
+`
+
+export const ChevronWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 3px;
```

Comment:
```suggestion
  margin-left: 2px;
```
홀수로 된 px는 최대한 지양하고 있고, 피그마에서 올바른 위계를 택하지 않았거나, 디자이너의 실수일 확률이 대부분입니다.
![image](https://user-images.githubusercontent.com/33861398/152975510-214c6d9a-6623-475d-bb88-cb7c889f80d3.png)